function fastifyDoneError(err?: Error): void {
  console.log(err);
}

export { fastifyDoneError };
